frontend/packages/integration-tests-cypress/tests/storage/snapshot.spec.ts

```diff
@@ -0,0 +1,85 @@
+import { testName, checkErrors } from '../../support';
+import {
+  SnapshotDetails,
+  snapshotClassDropdown,
+  dropdownFirstItem,
+  pvcDropdown,
+} from '../../views/snapshot';
+import { listPage } from '../../views/list-page';
+import { PVC, testerDeployment, SnapshotClass } from '../../mocks/snapshot';
```

Comment:
Hi, Can this be handled by the standard [Cypress Fixtures](https://docs.cypress.io/api/commands/fixture.html) api?